src/xcode/ENA/ENA/Source/HTTPClient/Resources/OTPAuthorizationForELS/OTPResponsePropertiesReceiveModel.swift

```diff
@@ -0,0 +1,10 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+struct OTPResponsePropertiesReceiveModel: Codable {
```

Comment:
Shouldn't it be just OTPReceiveModel

Reply:
there is another OTP, EDUS, so i wanted to be specific